infra/test/Test/Pos/Infra/Json.hs

```diff
@@ -151,15 +150,7 @@ golden_TopologyTraditional =
 
 roundTripTopology :: Property
 roundTripTopology =
-    eachOf 1000 genTopology roundTripsAesonShow
-
-roundTripTopologyYaml :: Property
-roundTripTopologyYaml =
-    eachOf 1000 genTopology (\a -> tripping a Y.encode unsafeDecode)
-  where
-    unsafeDecode a = case Y.decodeEither' a of
-      Left _  -> Nothing
-      Right b -> Just b
+    roundTripsAesonYamlShow 1000 genTopology
 
```

Comment:
It seems due to the issue of `ParseException` not having an easily derivable `Eq` instance, the error was being thrown away as a fix.